boom-app/src/components/DisplayComponent.vue

```diff
@@ -0,0 +1,41 @@
+<script setup lang="ts">
```

Comment:
In principe is alles in vue een component. De `View` suffix is wel handig om snel te zien dat iets een view is in de router, maar ik zou de Component suffix weglaten. Display vind ik ook iets te generiek. Misschien iets in de trant van CsvOutputTable?